### DISCUSSION TITLE
建议：是否可以添加一些示例呢？

### DISCUSSION THREAD

**Discussion by ymmz** [note286/xduts#19] [Ideas]
请问是否可以添加一些示例呢？比如说插入算法图片表格，排版的示例。
我是先看的旧模板，有很多参考，在新模板里面配置起来的确很方便，但是写的时候对小白不太友好，没有可以参考写法。
我试着写了一下，算法和图片需要引入宏包才行，而且格式报错了。
之前的模板虽然比较繁琐，但是对小白真的很友好。

**Comment by note286** [comment]
暂时不考虑添加。

原因：

- XDUTS不是LaTeX教程，而是学位论文模板，实现的是学校要求的格式。
- 如果你需要学习怎么绘制图、表和算法等，你应该看LaTeX入门教程（我在PDF文档里已经提及并给出了文档链接）和对应宏包的文档。
- 如果你是第一次使用LaTeX，我给你示例也没多大帮助，你还是需要自己琢磨琢磨；如果你不是第一次使用，由于每个人的使用习惯不同，都有个人偏好的宏包和写法，我给了参考意义也不大。
- 我之所以没有移除旧库，是因为我移除了也会有人找盗版使用，所以索性就放那吧。

你可以在睿思私信我，加入内测群交流。

另外，你所描述的不属于bug，我在PDF文档里写的很清楚，你应该在讨论区里交流。

**Comment by note286** [comment]
请参考#40。
